rtos/Thread.h

```diff
@@ -54,7 +54,7 @@ namespace rtos {
  *
  *  // Spawns a thread to run blink for 5 seconds
  *  int main() {
- *      thread.start(led1, blink);
+ *      thread.start(callback(blink, &led1));
  *      Thread::wait(5000);
```

Comment:
Note, should be `wait_ms(5000)` (or `wait(5)`).

Also [line 51](https://github.com/ARMmbed/mbed-os/pull/3469/files#diff-d80b651479def5286efa7efc857048cdL51)